src/components/Sender.tsx

```diff
@@ -125,6 +127,7 @@ const Sender = () => {
   const AccountSelected = () => {
     if (account) {
       const text = getName(account);
+      <AccountDisplay friendlyName={text} address={account.address} balance={senderAccountBalance?.formattedBalance} />;
```

Comment:
Im pretty sure this is some kind of copy/paste issue? This code does not make any sense here

Reply:
ooouuhhh!!!